knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -335,9 +336,9 @@ func (r *ReconcileKnativeKafka) transform(manifest *mf.Manifest, instance *serve
 		configureEventingKafka(instance.Spec),
 		ImageTransform(common.BuildImageOverrideMapFromEnviron(os.Environ(), "KAFKA_IMAGE_")),
 		socommon.VersionedJobNameTransform(),
-		socommon.ConfigMapVolumeChecksumTransform(context.Background(), r.client, dependentConfigMaps),
-		rbacProxyTranform,
-	)
+		socommon.ConfigMapVolumeChecksumTransform(context.Background(), r.client, sets.NewString("config-tracing", "kafka-config-logging")),
+		rbacProxyTranform), socommon.DeprecatedAPIsTranformersFromConfig()...)
```

Comment:
`DeprecatedAPIsTranformersFromConfig` ?

Reply:
DeprecatedAPIsTranformersFromVersion would be more accurate the idea is to check the K8s version and return the right transformers.